client/my-sites/stats/annual-highlights-section/style.scss

```
@@ -1,15 +1,11 @@
 @import "@automattic/typography/styles/variables";
 
-.stats-period-navigation.stats-year-navigation {
-	.stats-period-navigation__previous,
-	.stats-period-navigation__next {
-		&:focus,
-		&:active {
-			outline: thin dotted;
-		}
-	}
+.highlight-year-navigation {
+	.stats-navigation-arrows {
+		margin: 0 0 16px;
 
-	@media ( min-width: 661px ) {
-		padding-right: 0;
+		@media ( max-width: 661px ) {
```

```suggestion
		@media ( max-width: $custom-mobile-breakpoint ) {
```

This breakpoint could be related to the [highlight-cards variable](https://github.com/Automattic/wp-calypso/blob/trunk/packages/components/src/highlight-cards/variables.scss#L6).

---

Thanks for the heads up. I'll get this updated.